### PR TITLE
[Docs] PR for Add steps to install systemd related DB service unit files in the YB anywhere document

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-3837

Preview: https://deploy-preview-12377--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises/#install-systemd-related-database-service-unit-files

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md [hunk 1]
```diff
@@ -514,7 +515,257 @@ If YugabyteDB Anywhere will be using **cron jobs**, make sure the yugabyte user
...
+   /bin/systemctl restart yb-collect_metrics, \
+   /bin/systemctl enable yb-collect_metrics, \
+   /bin/systemctl disable yb-collect_metrics, \
+   /bin/systemctl daemon-reload
+   ```
+
+   <br>Open the `sudoers` file located in the `/etc` directory and add the following line to place the `yugabyte` file at the top-level `sudoers` file:
+   
+     ```sh
+   includedir /etc/sudoers.d
```
Comment (chidmuthu) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md:573]: This needs a '#' in front. It is not a comment in this context that's just the sudoers syntax

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md [hunk 2]
```diff
@@ -514,7 +515,257 @@ If YugabyteDB Anywhere will be using **cron jobs**, make sure the yugabyte user
...
+     ```
+   
+2. Ensure that you have the root access and add the following service and timer files to the `/etc/systemd/system` directory. Set their ownerships to the `yugabyte` user and 0644 permissions:
```
Comment (chidmuthu) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md:576]: Probably no need for "the"?

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md [hunk 3]
```diff
@@ -514,7 +515,257 @@ If YugabyteDB Anywhere will be using **cron jobs**, make sure the yugabyte user
...
+
+   ```sh
+   yb-master.service
```
Comment (chidmuthu) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md:579]: This is the name of the file and the contents start with the line 
[Unit]

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md [hunk 4]
```diff
@@ -514,7 +515,257 @@ If YugabyteDB Anywhere will be using **cron jobs**, make sure the yugabyte user
...
 -->
 
-You have finished configuring your on-premises cloud provider. Proceed to [Configure the backup target](../../backup-target/), or [Create deployments](../../../create-deployments/).
+You have finished configuring your on-premises cloud provider. Proceed to [Configure the backup target](../../backup-target/) or [Create deployments](../../../create-deployments/).
+
+##### Install systemd-related database service unit files
+
+As an alternative to setting crontab permissions, you can install systemd-specific database service unit files, as follows:
+
+1. Obtain sudo root access and add the `yugabyte` file to `/etc/sudoers.d/`. Next, open the `/etc/sudoers.d/yugabyte` file and write the following to provide permissions for the `yugabyte` user to run systemctl commands as a sudo user in order to manage the database:
```
Comment (chidmuthu) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md:524]: I think for this first step, we can just be a bit more general. We can say "Provider the yugabyte user the ability to run the following commands" and then have this shell block. Then, we can remove the specifications of /etc/sudoers.d/ and the includedir.

**Review by chidmuthu — CHANGES_REQUESTED** [review]
Left some small comments. Feel free to ping me if they are confusing.

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md:576]: fixed

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md (hunk 3 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md:579]: Ahh... I wasn't sure.
Fixed.

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md (hunk 4 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md:524]: Changed

**Comment by lizayugabyte** [comment]
@chidmuthu I implemented review comments. Could you take a quick look, please?

**Comment by lizayugabyte** [comment]
All the review comments have been implemented. I'll close the PR as "done".

**Pull request merged**